src/Chainweb/Version.hs

```diff
@@ -839,6 +840,13 @@ enableModuleNameFix2 Mainnet01 bh = bh >= 752214 -- ~ 2020-07-17 0:00:00 UTC
 enableModuleNameFix2 Testnet04 bh = bh >= 289966 -- ~ 2020-07-13
 enableModuleNameFix2 _ bh = bh >= 2
 
+-- | Enable serialization of Pact Events (Pact 3.6) in output
+enablePactEvents :: ChainwebVersion -> BlockHeight -> Bool
+enablePactEvents Mainnet01 bh = bh >= 1_013_500 -- 2020-10-15 18:47:13 UTC
```

Comment:
@larskuhtz @mightybyte please confirm

Reply:
Passes my rough sanity check.  @larskuhtz should probably put it into his more detailed calculation.